client/src/components/forum/ForumTopicThreads.jsx

```diff
@@ -7,12 +7,14 @@ import Footer from "../footer/Footer";
 import BackButton from "./BackButton";
 
 const ForumTopicThreads = () => {
-  // trying to connect to the DB
   const { topic } = useParams();
   const [threads, setThreads] = useState([]);
   const [error, setError] = useState(null);
-  console.log(topic);
```

Comment:
I know that for production we should remove debugging but maybe for our project submission we could just comment out to show we've used it? Thoughts? Happy either way

Reply:
happy to remove!! will do this on my next push :)